openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataInsightSettings.spec.ts

```diff
@@ -0,0 +1,220 @@
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import test, { expect } from '@playwright/test';
+import {
+  customFormatDateTime,
+  getCurrentMillis,
+  getEpochMillisForFutureDays,
+} from '../../../src/utils/date-time/DateTimeUtils';
+import { GlobalSettingOptions } from '../../constant/settings';
+import {
+  getApiContext,
+  redirectToHomePage,
+  toastNotification,
+} from '../../utils/common';
+import { settingClick } from '../../utils/sidebar';
+
+test.use({ storageState: 'playwright/.auth/admin.json' });
+
+test.describe.serial('Data Insight settings page should work properly', () => {
+  test.beforeEach(async ({ page }) => {
+    await redirectToHomePage(page);
+
+    const appsResponse = page.waitForResponse(
+      `/api/v1/apps?limit=15&include=non-deleted`
+    );
+
+    await settingClick(page, GlobalSettingOptions.APPLICATIONS);
+    await appsResponse;
+  });
+
+  test('Edit data insight application', async ({ page }) => {
+    const appResponse = page.waitForResponse(
+      `/api/v1/apps/name/DataInsightsApplication?fields=owners%2Cpipelines&include=all`
+    );
+
+    // Click on the config button
+    await page.click(
+      '[data-testid="data-insights-application-card"] [data-testid="config-btn"]'
+    );
+
+    await appResponse;
+
+    // Verify response status code (this is done implicitly by the route fulfillment)
+    // Click on the edit button
+    await page.getByTestId('edit-button').click();
+
+    // Select cron type
+    await page.getByTestId('cron-type').click();
+    await page.click('.rc-virtual-list [title="Day"]');
+
+    // Select hour
+    await page.click('[data-testid="hour-options"]');
+    await page.click('#hour-select_list + .rc-virtual-list [title="06"]');
+
+    // Select minute
+    await page.click('[data-testid="minute-options"]');
+    await page.click('#minute-select_list + .rc-virtual-list [title="00"]');
+
+    // Click on deploy button
+    await page.click('.ant-modal-body [data-testid="deploy-button"]');
+
+    await toastNotification(page, 'Schedule saved successfully');
+
+    // Verify response status code (this is done implicitly by the route fulfillment)
+    // Verify cron string
+    const cronString = await page
+      .locator('[data-testid="cron-string"]')
+      .textContent();
+
+    expect(cronString).toContain('At 06:00 AM');
+  });
+
+  test('Uninstall application', async ({ page }) => {
+    const appResponse = page.waitForResponse(
+      `/api/v1/apps/name/DataInsightsApplication?fields=owners%2Cpipelines&include=all`
```

Comment:
```suggestion
    const appResponse = page.waitForResponse(
      `/api/v1/apps/name/DataInsightsApplication?fields=*`
```
prevent flakiness for future